src/org/armedbear/lisp/Lisp.java

```diff
@@ -54,6 +54,72 @@
 
   public static boolean initialized;
 
+  // stepping related attributes
```

Comment:
@alejandrozf Please move all the "new" code for the stepper to the end of `Lisp.java`.  Code that changes existing functions is fine to modify "in-place".  

This non-functional change will help in looking through the history of `Lisp.java` with tools that compare lines directly without additional heuristics (like `patch` and `diff`).